stdlib/LinearAlgebra/test/lu.jl

```diff
@@ -341,4 +337,13 @@ end
     @test F.p == []
 end
 
+@testset "Issue #33547, condition number of 2x2 matrix" begin
+    M = [1.0 -2.0; -2.0 -1.5]
+    @test cond(M, 1) ≈ 2.227272727272727
+end
+@testset "Issue #33297, condition number of singular matrix" begin
+    M = [1. 1.; 0. 0.]
+    @test cond(M, 1) == Inf
+end
+
```

Comment:
I have added my tests to the test set "Matrix condition number" in `stdlib/LinearAlgebra/test/dense.jl`. In my opinion this is a better place for these tests, because then they're bundled.

Maybe we could add this:
```julia
for p in (1, 2, Inf)
        @test cond(zeros(Int, 2, 2), p) == Inf
        @test cond(zeros(2, 2), p) == Inf
        @test cond([0 0; 1 1], p) == Inf
        @test cond([0. 0.; 1. 1.], p) == Inf
end